Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN set -x && \
 FROM --platform=$BUILDPLATFORM base-builder AS coredns-builder
 ARG SRC=github.com/coredns/coredns
 ARG PKG=github.com/coredns/coredns
-ARG TAG=v1.12.2
+ARG TAG=v011
```

Comment:
This is an obsolete release:

https://github.com/coredns/coredns/releases/tag/v011

Reply:
https://github.com/rancher/image-build-coredns/pull/106